lib/util/security/tls.go

```diff
@@ -34,7 +34,7 @@ import (
 	"go.uber.org/zap"
 )
 
-const DefaultCertExpiration = 10 * 365 * 24 * time.Hour
+const DefaultCertExpiration = 72 * time.Hour
```

Comment:
Why 72 hours?

Reply:
The original default value will be trouble some to represent in `hours`. And the maximum duration unit is  `hr`.

The expiration time will be extremely for 10 years, which is scary on the first sight of `proxy.toml`.

Considering, you can now specify the config by yourself and auto-certs should not be used in serious cases, it is OK to me.

Reply:
We need a reasonable value for default values, otherwise it's senseless. I think 3 months is a reasonable time.

Reply:
Makes sense to me,